### PR TITLE
[quant] add serialization method for quantized hardswish

### PR DESCRIPTION
**Pull request by leslie-fang-intel**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #94486

**Summary**
Fix the issue: https://github.com/pytorch/pytorch/issues/91877. The root cause is serialization and deserialization method for `state_dict` does not enable for `QuantizedHardswish`. Added these methods in this PR.

**Test plan**
```
python -m pytest quantization/core/test_quantized_module.py -k test_hard_swish
```

cc @jgong5 @mingfeima @XiaobingSuper @sanchitintel @ashokei @jingxu10 @Xia-Weiwen

### CODE REVIEW

**Review by jgong5 — APPROVED** [review]

**Comment by leslie-fang-intel** [comment]
@supriyar Could you kindly help to take a look of this PR?

File: torch/ao/nn/quantized/modules/activation.py [hunk 1]
```diff
@@ -68,6 +68,23 @@ def forward(self, input):
     def _get_name(self):
         return 'QuantizedHardswish'
 
+    # ===== Serialization methods =====
```
Comment (vkuzo) [inline torch/ao/nn/quantized/modules/activation.py:71]: thanks for the fix!  To be consistent with other similar modules, could we do this by saving scale and zero_point as buffers intead?  https://github.com/pytorch/pytorch/blob/f172c7c60a5890136ea5f6225a56e41cb5d6e38f/torch/ao/nn/quantized/modules/normalization.py#L21 is an example.

Comment (leslie-fang-intel) [reply torch/ao/nn/quantized/modules/activation.py:71]: @vkuzo Changed and thanks for the suggestions. Please help to take a look again.

**Review by vkuzo — APPROVED** [review]
thanks for the fix!
